internal/gatewayapi/testdata/securitypolicy-with-jwt.in.yaml

```diff
@@ -86,11 +86,11 @@ securityPolicies:
         - header: one-route-example-key
           claim: claim1
       - name: example2
-        issuer: https://two.example.com
+        issuer: http://two.example.com
```

Comment:
we shouldn't keep the test cases to `https` since thats the default case imo

Reply:
There're two jwks in this test file, the first one is `https`, the changed one is the second.